db/test/versions/0051_test.js

```diff
@@ -0,0 +1,6 @@
+const testing = require('taskcluster-lib-testing');
+
+suite(testing.suiteName(), function() {
+  // this version only creates a method, which is tested in unit tests,
+  // so there is nothing to test on upgrade
+});
```

Comment:
Is this annoying, or helpful as forcing you to consider whether the revision needs tests?

Reply:
I don't mind either way. I just copy-paste the last method-only `_test.js`!